src/tyr/height_serializer.cc

```diff
@@ -94,6 +94,11 @@ std::string serializeHeight(const Api& request,
     json->emplace("id", request.options().id());
   }
 
+  // add warnings to json response
+  if (request.info().warnings_size() >= 1) {
+    json->emplace("warnings", serializeWarnings(request));
+  }
```

Comment:
`"warnings"` is only returned if there was at least one warning, so it's conditionally present in JSON output. I'd personally prefer to output an empty array, I think it's nicer for end users in most scenarios. Other opinions?

Reply:
Yeah thanks for bringing this up. I've done some more work with APIs and this is definitely a good way to go as some users may write their logic keeping in mind there's a warmings output and if not properly configures for null safety could be problematic. 
I'll update the PR. I certainly have a lot to catch up on.  😁

Reply:
no worries, I just carried it over the finish line, wasn't much to left to do:)

Reply:
Great, thanks.